Phonebook.Frontend/src/app/pages/dashboard/dashboard-routing.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { DashboardComponent } from 'src/app/pages/dashboard/components/overview/dashboard.component';
+import { BookmarkedComponent } from 'src/app/pages/dashboard/components/bookmarked/bookmarked.component';
+import { TeamComponent } from 'src/app/pages/dashboard/components/team/team.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: DashboardComponent,
+    children: [
+      { path: 'bookmarked', component: BookmarkedComponent, pathMatch: 'full' },
+      {
+        path: 'team',
+        component: TeamComponent
+      }
+    ]
+  },
+  { path: '**', redirectTo: '' }
```

Comment:
This causes a blank screen
```suggestion
  { path: '**', redirectTo: 'bookmarks' }
```